tests/dummy/config/targets.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-env node */
+module.exports = {
+  browsers: [
+    'ie 9',
```

Comment:
is it still IE9?

Reply:
Yea, I think we merge this & do a major version bump if we want to change which browsers we support (which is perfect timing with Ember 3.x :D)